Backend.Tests/Controllers/AudioControllerTests.cs

```diff
@@ -92,17 +93,28 @@ public void TestUploadConsentNullFile()
         }
 
         [Test]
-        public void TestUploadConsentEmptyFile()
+        public void TestUploadAudioFileEmptyFile()
         {
             // Use 0 for the third argument
             _file = new FormFile(_stream, 0, 0, "Name", FileName);
 
             var result = _audioController.UploadAudioFile(_projId, _wordId, _file).Result;
             Assert.That(result, Is.InstanceOf<BadRequestObjectResult>());
+
             result = _audioController.UploadAudioFile(_projId, _wordId, "speakerId", _file).Result;
             Assert.That(result, Is.InstanceOf<BadRequestObjectResult>());
         }
 
+        [Test]
+        public void TestUploadAudioFileNoWord()
+        {
+            var result = _audioController.UploadAudioFile(_projId, "not-a-user", _file).Result;
```

Comment:
I believe this was the intention.
```suggestion
            var result = _audioController.UploadAudioFile(_projId, "not-a-word", _file).Result;
```